client/src/index.css

```diff
@@ -63,3 +63,15 @@
     @apply mx-auto;
   }
 }
+
+
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+body {
```

Comment:
Is it possible to remove the default css of the body as flex? I think we can just add flex whenever we want to and not make it as default behavior? I feel like it might cause some bugs in the future